### DISCUSSION TITLE
[@types/node] `diagnostics_channel` types differ from Node.js docs

### DISCUSSION THREAD

**Discussion by nitreojs** [DefinitelyTyped/DefinitelyTyped#59927] [Issues with a @types package]
First of all, `node:diagnostics_channel` is a great package and I want to use it in my projects (even though it's experimental). But there are biiig problems with `@types/node` for `node:diagnostics_channel`:
- [`Channel`](https://nodejs.org/api/diagnostics_channel.html#class-channel) does not have [`publish(message: any)`](https://nodejs.org/api/diagnostics_channel.html#channelpublishmessage) method at all
- `ChannelListener` has its params in wrong order, in [docs](https://nodejs.org/api/diagnostics_channel.html#channelsubscribeonmessage) it's `(message: any, name: string | Symbol) => void`. Currently it is `(name: string, message: unknown) => void` 🤷

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
I'm fixing params order here: #59796
